### DISCUSSION TITLE
blog/low-profile-wireless-corne-keyboard/

### DISCUSSION THREAD

**Discussion by giscus** [armno/armno.in.th#47] [Blog Post Comments]
# blog/low-profile-wireless-corne-keyboard/

Notes from building my wireless Corne keyboard with low-profile Kailh Choc v1 switches

https://armno.in.th/blog/low-profile-wireless-corne-keyboard/

**Comment by Tronikart** [comment]
Thank you for the post! I'm currently trying out the alice layout with an ABM006 but the fact that it has no QMK is a bit of a deal breaker for me, this post made the whole process of building a Corne feel less dauting (as I have no experience soldering), so might end up returning this and making a Corne!

**Comment by Poof103** [comment]
I need help so this project i dont want to do any soldering so i was wondering if there a version where ther are hotspaw sockets on the keyboard and a way to order them to solder on both sides of the pcb on jlcpcb.

**Reply by Poof103** [reply]
i forgot and also the micro controler on the gerber file already

**Reply by armno** [reply]
Hi there. I'm not sure if there is already a PCB with hotswap sockets pre-installed available from JLCPCB. Maybe you can check splitkb.com or https://shop.beekeeb.com/ if they have some.
